lib/announcements.js

```diff
@@ -0,0 +1,152 @@
+  if (!article) return {
+    notFound: true
+  }
+
+  const path = join('announcements', article.file)
+  const mdxPathAbsolute = join(process.cwd(), 'content', path)
+  const mdxRawContent = await getRawFile(path)
+  const { content, data } = matter(mdxRawContent)
+  if (!data.title) {
+    data.title = ''
+  }
+  
+  const slugger = new GithubSlugger()
+  const mdxSource = await serialize(
+    new VFile({
+      value: content,
+      path: mdxPathAbsolute
+    }),
+    {
+      parseFrontmatter: false,
+      scope: { data },
+      mdxOptions: {
+        mdExtensions: [],
+        mdxExtensions: ['.md', '.mdx'],
+        rehypePlugins: [
+          rehypeSlug,
+          [
+            rehypeAutolink,
+            {
+              behavior: 'append',
+              properties: {
+                className: ['btn-copy-link', 'invisible']
+              },
+              content: {
+                type: 'element',
+                tagName: 'svg',
+                properties: {
+                  className: ['h-6', 'w-6', 'ml-2', 'docs-copy-btn'],
+                  xmlns: 'http://www.w3.org/2000/svg',
+                  fill: 'none',
+                  viewBox: '0 0 24 24',
+                  stroke: 'currentColor'
+                },
+                children: [
+                  {
+                    type: 'element',
+                    tagName: 'path',
+                    properties: {
+                      strokeLinecap: 'round',
+                      strokeLinejoin: 'round',
+                      strokeWidth: 2,
+                      d: 'M13.828 10.172a4 4 0 00-5.656 0l-4 4a4 4 0 105.656 5.656l1.102-1.101m-.758-4.899a4 4 0 005.656 0l4-4a4 4 0 00-5.656-5.656l-1.1 1.1'
+                    },
+                    children: []
+                  }
+                ]
+              }
+            }
+          ]
+        ],
+        remarkPlugins: [
+          remarkHeadingId,
+          remarkGfm,
+          remarkCodeImport,
+          [
+            remarkRewriteImages,
+            { destination: process.env.ASSETS_DESTINATION }
+          ],
+          remarkInlineLinks,
+          [remarkExternalLinks, { target: false, rel: ['nofollow'] }],
+        ]
+      },
+      target: ['esnext']
+    }
+  )
+  
+  const markdownTokens = marked.lexer(content)
+  const headings = markdownTokens
+    .filter((t) => t.type === 'heading')
+    .map((heading) => {
+      heading.slug = slugger.slug(heading.text)
+      return heading
+    })
+
+  const firstHeadingText =
+    headings && headings.length > 0 ? headings[0].text : ''
+  const title = data.title.length > 0 ? data.title : firstHeadingText
+
+  return {
+    title,
+    date: article.date.toDateString(),
+    frontmatter: data,
+    source: mdxSource,
+    tocHeadings: headings,
+  }
+}
+
+function getSlug(date, slug) {
+  return [
+    '' + date.getFullYear(),
+    ('0' + date.getMonth()).slice(-2),
```

Comment:
```suggestion
    ('0' + (date.getMonth()+1)).slice(-2),
```
A long time ago, I actually created a stackoverflow answer for this issue: https://stackoverflow.com/a/34290167